static_data/completeness_test.csv

```diff
@@ -22,7 +22,7 @@ Step 3: Nationally downloaded data,SRTM 30m elevation,*,elev,dem,ras,
 ,WorldPop 1km,*,popu,pop,ras,
 ,WorldPop 100m,,popu,pop,ras,
 ,WorldPop sex & age disaggregated,*,popu,pop,tab,
-,HDX COD (or other) population,,,,,
+,HDX COD (or other) population,*,popu,pop,*,
```

Comment:
should at the end be *? there are tab, pt, py and etc at other points

Reply:
Yes, there can be *. Population can be either TIFF (ras) or CSV (tab),